lib/std/c/solaris.zig

```diff
@@ -9,6 +9,12 @@ const timezone = std.c.timezone;
 extern "c" fn ___errno() *c_int;
 pub const _errno = ___errno;
 
+// TODO: Add sysconf numbers when the other OSs do.
+pub const _SC = struct {
+    pub const NPROCESSORS_ONLN = 15;
+    // TODO PAGESIZE
```

Comment:
[It's 11](https://github.com/illumos/illumos-gate/blob/69764de32bf29a19d2a8b8832adafc4bd4f934f4/usr/src/uts/common/sys/unistd.h#L117C1-L117C1).

Reply:
I'm looking for all constants:
```
#!/usr/bin/env sh

git clone --depth=1 https://github.com/DragonFlyBSD/DragonFlyBSD
git clone --depth=1 https://github.com/freebsd/freebsd-src
git clone --depth=1 https://fuchsia.googlesource.com/fuchsia
git clone --depth=1 https://github.com/haiku/haiku
git clone --depth=1 https://github.com/hermitcore/rusty-hermit
git clone --depth=1 https://github.com/Stichting-MINIX-Research-Foundation/minix/
git clone --depth=1 https://github.com/NetBSD/src NetBSD
git clone --depth=1 https://github.com/openbsd/src openbsd
git clone --depth=1 https://github.com/kofemann/opensolaris
```
Download takes time though, so I think I'm sleeping now and finish tomorrow.

Reply:
Finished. Take a look at https://github.com/ziglang/zig/pull/16638#issuecomment-1659957271 for how to get the bits. I'll patch up the upstream urls shortly. This takes much longer than planned.